opentelemetry-distro/src/opentelemetry/distro/__init__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+
+from opentelemetry.instrumentation.distro import BaseDistro
+
+
+class OpenTelemetryDistro(BaseDistro):
+    """
+    The OpenTelemetry provided Distro configures a default set of
+    configuration out of the box.
+    """
+
+    def _configure(self, **kwargs):
```

Comment:
What's the reason for separating out configuration between distros and configurators? Why can't we just have one class that controls all possible configuration?

Reply:
The reason for this being separate is that the `Configurator` implemented by the SDK is more generically configuring the providers by utilizing what is configured in the env variables. The `Distro` on the other hand lets users pre-configure those environment variables